### ISSUE TITLE
HTML display error on page SDL_EventType

### ISSUE THREAD

**Issue by Nicolas378** [libsdl-org/sdlwiki#76]
![image](https://user-images.githubusercontent.com/83796802/121583815-bf1a9a80-c9fe-11eb-9b3a-0aab570252a3.png)

As seen on Chrome

**Comment by icculus** [comment]
Yeah, this page needs to be fixed up for the new wiki still. I'll take care of this after 2.0.16 ships.

**Comment by pedro-w** [comment]
this is fixed, I think by f220cb632c56e9e8b820ec2f5309bbdf1a54702e
